x-pack/plugins/ml/public/components/anomalies_table/anomalies_table.js

```diff
@@ -202,11 +203,39 @@ function getColumns(
     }
   }
 
-  columns.push({
-    field: 'jobId',
-    name: 'job ID',
-    sortable: true
-  });
+  if (examplesByJobId && Object.keys(examplesByJobId).length > 1) {
+    columns.push({
+      field: 'jobId',
+      name: 'job ID',
+      sortable: true
+    });
+  }
+
+  const showExamples = items.some(item => item.entityName === 'mlcategory');
+  if (showExamples === true) {
+    columns.push({
+      name: 'category examples',
+      sortable: false,
+      truncateText: true,
+      render: (item) => {
+        const examples = _.get(examplesByJobId, [item.jobId, item.entityValue], []);
+        const categoryTabIndex = 1;
```

Comment:
Those numbers (`0` = Details / `1` = `Category Examples`) could come from an Enum, something like `const categoryTabIndex = CATEGORY_TAB.DETAILS`.

Reply:
Stored these in a constant 👍